tests/src/podman-machine-onboarding.spec.ts

```diff
@@ -0,0 +1,197 @@
+      await playExpect(podmanOnboardingPage.podmanMachineUserModeNetworkingCheckbox).not.toBeChecked();
+      await playExpect(podmanOnboardingPage.podmanMachineStartAfterCreationCheckbox).toBeChecked();
+    });
+  });
+  describe.skip('Podman Machine creation and operations', async () => {
+    test('Create a default Podman machine', async () => {
+      const createMachineButton: Locator = podmanMachineConfiguration.getByRole('button', { name: 'Create' });
+      await createMachineButton.click();
+      const expandLogsButton: Locator = podmanOnboardingPage.mainPage.getByRole('button', { name: 'Show Logs' });
+      await playExpect(expandLogsButton).toBeVisible();
+      await expandLogsButton.click();
+      const machineCreationStatusMessage: Locator =
+        podmanOnboardingPage.mainPage.getByLabel('Onboarding Status Message');
+      await playExpect(machineCreationStatusMessage).toBeVisible({ timeout: PODMAN_MACHINE_STARTUP_TIMEOUT });
+      await playExpect(machineCreationStatusMessage).toHaveText('Podman successfully setup');
+      await podmanOnboardingPage.nextStepButton.click();
+    });
+    describe('Podman machine operations', async () => {
+      let podmanMachineStatus: Locator;
+      let podmanMachineStartButton: Locator;
+      let podmanMachineRestartButton: Locator;
+      let podmanMachineStopButton: Locator;
+      let podmanMachineDeleteButton: Locator;
+      test('Open podman machine details', async () => {
+        dashboardPage = await navigationBar.openDashboard();
+        await playExpect(dashboardPage.mainPage).toBeVisible();
+        settingsBar = await navigationBar.openSettings();
+        await settingsBar.resourcesTab.click();
+        resourcesPage = new ResourcesPage(page);
+        const podmanResourcesSectionLocator: Locator = resourcesPage.getPage().getByLabel('podman', { exact: true });
+        await playExpect(podmanResourcesSectionLocator).toBeVisible();
+        const podmanConnections: Locator = podmanResourcesSectionLocator.getByRole('region', {
+          name: 'Provider Connections',
+        });
+        await playExpect(podmanConnections).toBeVisible();
+        const podmanMachineDetails: Locator = podmanConnections.getByRole('button', { name: 'Podman details' });
+        await playExpect(podmanMachineDetails).toBeVisible();
+        await podmanMachineDetails.click();
+        podmanMachineStatus = page.getByLabel('Connection Status Label');
+        await playExpect(podmanMachineStatus).toBeVisible();
+        const podmanMachineControls: Locator = page.getByRole('group', { name: 'Connection Actions' });
+        await playExpect(podmanMachineControls).toBeVisible();
+        podmanMachineStartButton = podmanMachineControls.getByRole('button', { name: 'Start', exact: true });
+        await playExpect(podmanMachineStartButton).toBeVisible();
+        podmanMachineRestartButton = podmanMachineControls.getByRole('button', { name: 'Restart' });
```

Comment:
These locator definitions should be in the corresponding page object class

Reply:
done.